src/components/Navbar.js

```diff
@@ -0,0 +1,18 @@
+function Navbar() {
+  return (
+    <div className="nav-bar">
+      <div className="nav-con">
+        <nav className="nav-link">
+          <h1><a className="logo" href="/">Bookstore </a></h1>
+          <li><a href="/">Books</a></li>
+          <li><a href="/categories">Catagories</a></li>
+        </nav>
+        <div className="user">
+          <img src="./src/img/logo" alt="user" />
+        </div>
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] Well done :clap: you did a lovely job :+1: but please use the `<Link/>` component which is provided by `React router Dom` because when we use the `<Link/>` component your page will be redirected to the specific router without refreshing :+1: check below :point_down: my suggestion :slightly_smiling_face: 

```suggestion
import {Link} from 'react-router-dom'
function Navbar() {
  return (
    <div className="nav-bar">
      <div className="nav-con">
        <nav className="nav-link">
          <h1><Link className="logo" to="/">Bookstore </a></h1>
          <li><Link to="/">Books</a></li>
          <li><Link to="/categories">Catagories</a></li>
        </nav>
        <div className="user">
          <img src="./src/img/logo" alt="user" />
        </div>
      </div>
    </div>
  );
}
```